Formula/openjdk@8.rb

```diff
@@ -49,15 +49,16 @@ class OpenjdkAT8 < Formula
     depends_on "libxtst"
   end
 
-  # Oracle doesn't serve JDK 7 downloads anymore, so use Zulu JDK 7 for bootstrapping.
+  # Oracle doesn't serve JDK 8 downloads anymore, so we use Zulu JDK 8 for bootstrapping.
```

Comment:
Are we sure we want to switch to JDK 8 for bootstrapping? AFAIR it's recommended to use the previous version for this.

Reply:
There is no consistency to the bootstrapping right now. `openjdk@11` was using a mix of 10 and 11. `openjdk@19` is using 19.

Trying to bring all our `openjdk` formulae into a standard format.

Reply:
> openjdk@11 was using a mix of 10 and 11. 

I guess it's because OpenJDK 10 doesn't support ARM for bootstrapping OpenJDK 11. Probably we could switch to Zulu 10, as well.

> openjdk@19 is using 19.

I think it uses [`18.0.2.1`](https://github.com/Homebrew/homebrew-core/blob/b424abf145ba2094c12ef17863a38318ad8c2d98/Formula/openjdk.rb#L56-L77).

Ref https://openjdk.org/groups/build/doc/building.html#boot-jdk-requirements

Reply:
Thanks for the documentation. I was not aware of the options.
>The rule of thumb is that the boot JDK for building JDK major version N should be a JDK of major version N-1, so for building JDK 9 a JDK 8 would be suitable as boot JDK. However, the JDK should be able to "build itself", so an up-to-date build of the current JDK source is an acceptable alternative.

This looks to me like a "You can build it with itself, but it's preferred to do N-1". Do you get the same read on that?